lib/gzr/commands/dashboard.rb

```diff
@@ -53,7 +53,9 @@ def mv(dashboard_id, target_folder_id)
       method_option :transform,  type: :string,
                            desc: 'Fully-qualified path to a JSON file describing the transformations to apply'
       method_option :simple_filename, type: :boolean,
-                    desc: 'Use simple filename for output (Dashboard_<id>.json)'
+                           desc: 'Use simple filename for output (Dashboard_<id>.json)'
+      method_option :trim, type: :boolean,
+                           desc: 'Trim output to minimal set of fields for later import'
```

Comment:
does the trim toggle default to on or off?

Reply:
It is currently default off.